_posts/elements/2015-04-15-icons.md

```diff
@@ -0,0 +1,738 @@
+---
+layout:       element
+title:        Icons
+date:         2015-04-15 00:00:00
+
+categories:   elements
+tags:
+- atomic
+- pattern
+- icons
+
+slug:         icons
+scssurl:      ""
+
+description:  ...
+
+info: ...
+---
+
+<h3 class="hd-6">Media</h3>
+<div class="example-set">
+    <div class="swatch">
+        <div class="icon-display">
+            <svg class="icon icon-video" title="User account"><use xlink:href="{{ site.baseurl }}public/pl/images/edx-svg/svgdefs.svg#icon-video"></use></svg>
+        </div>
+        <div class="icon-classname">
```

Comment:
NIT: this is another place where we should sync up general classes used for examples vs. specifics needed per element demo. Here are some suggestions based on different types:

```html
<!-- basic example -->
<div class="example-display"> ... </div>
<div class="example-meta"> ... </div>

<!-- color example -->
<div class="example-display swatch-color brand-primary dark"> ... </div>
<div class="example-meta"> ... </div>

<!-- icon example -->
<div class="example-display"> ... </div>
<div class="example-meta icon-classname"> ... </div>
```